<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QTreeWidget" name="tw_cash_in_cassete">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Сервер</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Порт:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit">
                 <property name="text">
                  <string>8088</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pb_server_run">
                 <property name="text">
                  <string>Run</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="maximumSize">
                  <size>
                   <width>60</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Статус:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="l_server_status">
                 <property name="text">
                  <string>?</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Внесение</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="maximumSize">
                  <size>
                   <width>60</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Статус:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="l_deposit_status">
                 <property name="text">
                  <string>?</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pb_deposit_end">
                 <property name="text">
                  <string>Завершить</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Лимит:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="le_deposit_limit">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QPushButton" name="pb_deposit_10">
                 <property name="text">
                  <string>+10</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="pb_deposit_50">
                 <property name="text">
                  <string>+50</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="pb_deposit_100">
                 <property name="text">
                  <string>+100</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QPushButton" name="pb_deposit_200">
                 <property name="text">
                  <string>+200</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="pb_deposit_500">
                 <property name="text">
                  <string>+500</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="pb_deposit_1000">
                 <property name="text">
                  <string>+1000</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="pb_deposit_2000">
                 <property name="text">
                  <string>+2000</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QPushButton" name="pb_deposit_5000">
                 <property name="text">
                  <string>+5000</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="pb_engeneer">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Инженераный режим</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_run_inkas">
                <property name="text">
                 <string>Инкассация</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cb_close_door_1">
                <property name="text">
                 <string>Дверь 1 закрыта</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cb_close_door_2">
                <property name="text">
                 <string>Дверь 2 закрыта</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cb_cassete_input">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Касета вставлена</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cb_connect_bank">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Связь с банком</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QFrame" name="frame">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="l_display_line_1">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="l_display_line_2">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="l_display_line_3">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="l_display_line_4">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="la_status_sassion">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="l_status_level">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTextEdit" name="te_log"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
